network-api/networkapi/wagtailpages/customblocks/recent_blog_entries.py

```diff
@@ -0,0 +1,82 @@
+import json
+from django.apps import apps
+from wagtail.core import blocks
+
+
+class RecentBlogEntries(blocks.StructBlock):
+    title = blocks.CharBlock(
+        required=True,
+    )
+
+    tag_filter = blocks.CharBlock(
+        label='Filter by Tag',
+        required=False,
+    )
+
+    category_filter = blocks.ChoiceBlock(
+        label='Filter by Category',
+        required=False,
+        choices=[
+            ('all', 'All'),
+            ('Mozilla Festival', 'Mozilla Festival'),
+            ('Open Leadership & Events', 'Open Leadership & Events'),
+            ('Internet Health', 'Internet Health'),
+            ('Fellowships & Awards', 'Fellowships & Awards'),
+            ('Advocacy', 'Advocacy'),
+        ]
+    )
+
+    top_divider = blocks.BooleanBlock(
```

Comment:
If we really need these, we want to refactor this component and the pulse entry component to have a shared "parent" that does all this and we can get "for free" by extending, rather than duplicating it.

Reply:
ah yes, this makes complete sense 👍 ...but, what order would I do that in?

1. Create parent component
2. Extend class in recent/latest blog entry component
3. After 2 lands, refactor the other components that include the dividers and extend the class there?

Reply:
folding it into this PR should be fine, but if that would hold up the PR, we can land this first, with an immediate followup. (e.g. do the refactor while this PR is getting reviewed)